request-handler/src/main/java/au/org/aodn/aws/wps/RequestValidator.java

```diff
@@ -2,8 +2,6 @@
 
 import au.org.aodn.aws.wps.exception.ValidationException;
 
-import java.util.Properties;
-
 public interface RequestValidator {
-    void validate(AwsApiRequest request, Properties config) throws ValidationException;
+    void validate(AwsApiRequest request) throws ValidationException;
 }
```

Comment:
I'd leave this interface - the idea was that the properties could be rad from any source + passed in using the Properties object.

Reply:
We can read the config file within WpsConfig (from s3 or local file) and don't necessarily need to pass it from the RequestValidator. I have made some changes to the PR, so we can read the properties from any source.

Reply:
Why are we even modifying this config code?  What was there worked fine. The only change we needed to make was to source the values from environment variables - which could entirely be done in WpsConfig